src/libs/kata-types/src/utils/u32_set.rs

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
What is a "u32 set", why do we need it? What about u64/usize? etc. I'd consider adding a comment here to explain.

Reply:
How about this
```
/// A set of unique `u32` IDs.
///
/// The `U32Set` may be used to save CPUs parsed from a CPU list file or NUMA nodes parsed from
/// a NUMA node list file.
```